test/store/underlying/test_local_fs.py

```diff
@@ -0,0 +1,193 @@
+"""
```

Comment:
I appreciate your bringing in all of these tests. However, in my latest PR (that has since merged) I refactored tests for the underlying store representation to be parametric over the backend. That means that it should be possible to remove this file, add a fixture in `test/store/fixture.py`, and update the configuration in that file, and all of the existing artifact store tests should now execute for the filesystem implementation as well.

Reply:
Yes, to be honest, I noticed that they were the same tests for the other stores, and I remembered you had talked about making them parametric, but I guess I accidentally didn't merge that last version of master and I thought that was not done yet. I'll do that and update this.